src/components/rangeUsePlanPage/versionsList/VersionsDropDown.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useRef } from 'react'
+import useSWR from 'swr'
+import * as API from '../../../constants/api'
+import { axios, getAuthHeaderConfig } from '../../../utils'
+import VersionsDropDownList from './VersionsDropDownList'
+
+const sortVersions = (a, b) => {
+  if (b.version === -1) return 1
+  if (b.version > a.version) return 1
+  if (b.version < a.version) return -1
+  return 0
+}
```

Comment:
Preferably this sorting should be offloaded to the API. In any case though, we can remove the special sort for `-1`, since that's from a previous versioning implementation.

```suggestion
const sortVersions = (a, b) => {
  if (b.version > a.version) return 1
  if (b.version < a.version) return -1
  return 0
}
```

Reply:
Implemented here: 5bdaeec